codecparsers/bitReader_unittest.cpp

```diff
@@ -48,12 +48,12 @@ BITREADER_TEST(NoOverflow)
     std::vector<uint8_t>& data = largeData();
     const uint32_t nBytes = data.size();
 
-    ASSERT_EQ(0, nBytes % (1024 * 1024 * 512));
+    ASSERT_EQ(0u, nBytes % (1024 * 1024 * 512));
```

Comment:
`s` is not a constant, but 0u is.  If you change `s` type to `const unsigned int` then the warning goes away.